src/Cache/PPCache.cpp

```diff
@@ -205,16 +203,19 @@ bool PPCache::restore_(const fs::path& cacheFileName, bool errorsOnly) {
   return true;
 }
 
-bool PPCache::checkCacheIsValid_(const fs::path& cacheFileName) {
+bool PPCache::restore_(const fs::path& cacheFileName, bool errorsOnly) {
+  auto buffer = openFlatBuffers(cacheFileName);
+  if (buffer == nullptr) return false;
+
+  return restore_(cacheFileName, buffer, errorsOnly);
```

Comment:
similar to the valid check, if restore_() tests for null, this can simply be
```c++
return restore_(cacheFileName, openFlatBuffers(cacheFileName), errorsOnly);
```

Reply:
Done!